team.html

```diff
@@ -1,333 +1,472 @@
 
-        <br/>
-        <h3 id="Distribution_coordinator">Distribution
-          coordinator (1 per distribution channel)</h3>
-        <p>Create and maintain binary distribution packages for Gammapy core and
-          affiliated packages for a specific OS or package management system.</p>
--->
-         <hr>
-        <br/>
-        <h3 id="Release_coordinator">Release Coordinator</h3>
-        <p>The role of the Release Coordinator consists on taking care of the building of a gammapy release, installing it on the distribution forge and guaranting its compatibility on different OS.</p>
-
-         <hr>
-        <br/>
-        <h3 id="Subpackage_maintainer">Sub-package maintainer (1 per core package sub-package)</h3>
-        <p>The CC aims to assign a maintainer for each sub-package of the project.</p>
-        <p>The global role of a maintainer is to manage a sub-package of the gammapy core package, being responsible for its progress, its new features, including:</p>
+        <h2 id="core-developers">Core developers</h2>
+        <p class="lead">The Gammapy core developers regularly contribute to Gammapy, either by writing code, tests or
+            documentation themselves, or by doing review, maintenance, refactoring or cleanup for parts of the Gammapy
+            code.</p>
+        <p>
+            The Gammapy core developers are PhD students, postdocs or researchers with a permanent position in gamma-ray
+            astronomy that spend a certain fraction of their time on Gammapy. The role of Gammapy core developer isn't
+            well-defined, there is a continuous spectrum from Gammapy contributors that make one small pull request to
+            Gammapy, to Gammapy core developers that spend a large fraction of their work time on Gammapy.
+        </p>
+        <p>
+            Therefore, we do not try to draw a line here and present a list of all core developers.
+            However, to give some credit and to encourage contributors to help improve the quality of the existing
+            Gammapy code, tests and documentation, we do give a list here of the major Gammapy sub-packages,
+            and for each sub-package list a lead and where available deputy maintainer. The goal is to have
+            every sub-package covered, so if you are familiar with one of these sub-packages, and have some time
+            to work on it within the coming months, please sign up by contacting the Gammapy lead developer!
+        </p>
+        <p>
+            It is important to note that no-one "owns" a given sub-package or piece of code in Gammapy. Everyone is
+            welcome to make a proposal or pull request for any file in Gammapy at any time and it is an explicit goal
+            that all parts of the Gammapy codebase are easy to understand and contribute to by new people!
+        </p>
+        <p>
+            By agreeing to maintain a given sub-package for Gammapy, you express to do the following on a best-effort
```

Comment:
Sub-package maintainers ensure, on a best-effort basis, (roughly opted by priority) that:
- known bugs are fixed
- open PR with fixes and features are reviewed and integrated
- the existing code is well documented and tested.